tests/common/threads/test_update.py

```diff
@@ -80,7 +80,7 @@ async def test_update_profile_field(mocker, thread_dependencies):
     step_name, skip = await step.handle_emoji(raw_reaction)
     assert step_name is None
     assert skip is None
-    await assert_cache_metadata_content(user_id, cache, "field", "display_name")
+    assert_cache_metadata_content(user_id, cache, "field", "display_name")
```

Comment:
not sure if these drops are right. staging base has these since that function is async.

Reply:
I will add these back and merge